modules/javafx.graphics/src/main/java/javafx/css/converter/StringConverter.java

```diff
@@ -65,9 +68,16 @@ public String toString() {
         return "StringConverter";
     }
 
+    /**
+     * Converter to convert a sequence of {@code String}s to a String[].
```

Comment:
I recommend code style for the `String[]` or else say `an array of {@code String}s`.